leetcode/generate_md.py

```diff
@@ -1,2 +1,3 @@
 print('hello world!')
```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 Added zagadky 
 Mistakes: 
 Dude...it is not even in right format.